BigCommerce/Integration/01_ManageBigCommerceConnection.md

```diff
@@ -0,0 +1,195 @@
+# Manage BigCommerce connection
+
+*Actindo* features a special installation app in the *BigCommerce App Store*. The *Actindo Core1 DOP connector* app setups and authorizes a connection automatically. 
+Note, if possible, you should not delete the app in *BigCommerce* after creating the connection. Although the deleting will not delete the connection in *Omni-Channel* itself, but it will invalidate the access token you have created during connecting the app. If the app was deleted by mistake, either reconnect the app or enter a valid access token manually to enable the connection again.
+
+Note that you can use the *Actindo Core1 DOP connector* app for one connection only. If you have more than one store in *BigCommerce*, you can <!--- can oder must?-->create the first connection automatically with the *Actindo Core1 DOP connector* app. For all other stores, you must create the connection manually.
+
+
+## Create BigCommerce connection automatically 
+
+Create the connection using the *BigCommerce* driver. You will not be able to configure any other settings until the connection is established.
+
+
+### Prerequisites
+
+- You have a *BigCommerce* customer account to log in to *BigCommerce*.
+- You have created a *BigCommerce* store. 
+- In the *Actindo Core1 Platform*, you have permission to create connections in *Omni-Channel*.
```

Comment:
Braucht man wirklich eine spezielle Rolle/Genehmigung, um eine Verbindung erstellen zu können? Ich finde keine Option dazu in the ActindoUserGroup, s. Screenshot unten (aber ich bin in meinen Sandbox Administratorin). Bisher haben wir es bei den Verbindungen nicht erwähnt, daher bin ich unsicher, ob es so ist. Evtl. kannst du auch Oli dazu fragen.

Wenn ja, vielleicht ein Verweis hier auf die Core1-Doku (ManageUserGroup) hinzufügen?

![image](https://github.com/Actindo-AG/Documentation/assets/99268392/2f819b27-e804-4550-ba05-6b497f538e9a)